src/main/java/com/amazonaws/services/dynamodbv2/datamodeling/AttributeEncryptor.java

```diff
@@ -58,9 +64,20 @@ public DynamoDBEncryptor getEncryptor() {
     public Map<String, AttributeValue> transform(final Parameters<?> parameters) {
         // one map of attributeFlags per model class
         final ModelClassMetadata metadata = getModelClassMetadata(parameters);
+
+        final Map<String, AttributeValue> attributeValues = parameters.getAttributeValues();
+        if (metadata.doNotTouch) {
```

Comment:
This looks like a behavior change, unless I am missing something. Can we cover it?

Reply:
It isn't a behavior change. In a "DoNotTouch" case, we make no changes whatsoever to the underlying data. I'm just short-circuiting that logic.

Reply:
OK.